src/unity.yml

```diff
@@ -154,12 +158,16 @@ components:
       $ref: './unity/schemas/Me.yml'
     OrgUsers:
       $ref: './unity/schemas/OrgUsers.yml'
+    OperatorOrganization:
+      $ref: './unity/schemas/OperatorOrganization.yml'
+    OperatorOrganizations:
+      $ref: './unity/schemas/OperatorOrganizations.yml'
+    RelatedAccount:
```

Comment:
Should this stay where it was for 🔤  order?

Reply:
There didn't appear to be 🔤 order in the file, so that is why we didn't move it.